components/typography/test/typography.visual-diff.js

```diff
@@ -61,6 +61,11 @@ describe('d2l-typography', function() {
 			await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
 		});
 
+		it('blockquote', async function() {
```

Comment:
I should probably add an RTL for this.

Reply:
Yeah I think it would be worthwhile. In this case I think the fixture can just be set up with `<blockquote dir="rtl" ...>`.